builtin/providers/aws/resource_aws_dynamodb_table.go

```diff
@@ -343,10 +344,6 @@ func resourceAwsDynamoDbTableUpdate(d *schema.ResourceData, meta interface{}) er
 		return fmt.Errorf("Local secondary indexes can only be built at creation, you cannot update them!")
 	}
 
-	if d.HasChange("hash_key") {
-		return fmt.Errorf("Hash key can only be specified at creation, you cannot modify it.")
-	}
-
 	if d.HasChange("range_key") {
 		return fmt.Errorf("Range key can only be specified at creation, you cannot modify it.")
```

Comment:
We should probably apply the same fix here to `range_key` which I believe follows the same rule?